frontend/assets/styles/ant-design-dark.less

```diff
@@ -97,10 +97,5 @@
     display: block;
   }
 
-  .ant-alert-icon {
-    top: 16px;
-    left: 24px;
-  }
```

Comment:
Have you checked why was it added? Eg:
- Which PR added it
- Where else `Alert` component currently used